addons/survey/data/mail_template_data.xml

```diff
@@ -4,7 +4,7 @@
         <record id="mail_template_user_input_invite" model="mail.template">
             <field name="name">Survey: Invite</field>
             <field name="model_id" ref="model_survey_user_input" />
-            <field name="subject">Participate to {{ object.survey_id.display_name }} survey</field>
+            <field name="subject"/>
```

Comment:
As the rendering is not done anymore, we can probably keep the subject for people that want to use the template outside of the classic invite flow, isn't it ?